app/workers/pull_request_assigner_worker.rb

```diff
@@ -0,0 +1,32 @@
+module Pushbit
+  class PullRequestAssignerWorker < BaseWorker
+    def work(trigger_id)
+      trigger = Trigger.find(trigger_id)
+      payload = trigger.payload
+
+      repo_full_name = payload['repository']['full_name']
+
+      #TODO we need to use teams first
+      # Find possible assignees, filter our bot
+      collaborators = client.collaborators(repo_full_name).select { |c| c.login != ENV.fetch('GITHUB_BOT_LOGIN') }
+
+      puts payload['pull_request']['number']
+
+
+      # Choose one randomly
+      collaborator = collaborators.sample
+      assignee = collaborator.login
+
+      # Assign Issue
+      client.update_issue(repo_full_name, payload['pull_request']['number'], :assignee => assignee)
+
+      Task.create!({
+        behavior: "pull_request_assigner",
+        github_id: payload['pull_request']['number'],
+        repo_id: Repo.find_by!({
```

Comment:
Same as above, we can get the repo from the trigger association rather than querying manually